docs/metrics/metrics-operators/eval.md

```diff
@@ -34,15 +34,15 @@ metrics query | eval expr([REDUCER BOOLEAN EXPRESSION | _value] [_granularity])
 This query returns the value of the cpu_idle metric, multiplied by 100.
 
 ```
-_sourceCategory=ApacheHttpServer metrics=cpu_idle | eval _value * 100
+_sourceCategory=ApacheHttpServer metric=cpu_idle | eval _value * 100
 ```
 
 **Example 2**
 
 This query sets the value of each point in a single time series to the average of all values in that time series.
 
 ```
-metrics query | eval avg
+metric=cpu | eval avg
```

Comment:
It would be good to unify the selector between examples, we have `metric=cpu_idle` and `metric=cpu_idle` , sumo host metrics have `metric=CPU_Idle` and we might want to be consistent with it